api/dbadapters/redshift.ts

```diff
@@ -68,12 +68,12 @@ export class RedshiftDbAdapter implements IDbAdapter {
     const statementWithExplain = `explain ${statement}`;
```

Comment:
i'm not clear on how this code can still be compiling. you should need to change `evaluate` to match the new interface (and then evaluate the query(s) according to the type that is passed in)

Reply:
Yeah pretty weird that that's allowed. Have updated to use new interface in other adapters, and added a throw if they're given things other than strings.

Reply:
With TODOs